content-security-policy/support/echo-policy-multiple.py

```diff
@@ -0,0 +1,28 @@
+def main(request, response):
+    headers = [("Content-Type", "text/html")]
+    if "policy" in request.GET:
+        headers.append(("Content-Security-Policy", request.GET["policy"].replace("8", ";")))
+    if "policy2" in request.GET:
+        headers.append(("Content-Security-Policy", request.GET["policy2"].replace("8", ";")))
+    message = request.GET["id"]
+    return headers, '''
+<!DOCTYPE html>
+<html>
+<head>
+    <title>This page enforces embedder's policies</title>
+    <style>
+        body {
+            background-color: maroon;
+        }
+    </style>
```

Comment:
If I remove this, then it seems like post message is not delivered on time for the check in `assert_iframe_with_csp`. So I have to either set a timeout or leave this here.